starter-code/tests/ClueSpec.js

```diff
@@ -42,13 +50,38 @@ describe('Pick a random mistery - pickMistery', function () {
     expect(typeof pickMistery).toBe('function');
   });
 
+  function pickMistery(){
+  var RandomCharacter = randomSelector(charactersArray);
+  var RandomRoom = randomSelector(roomsArray);
+  var RandomWeapon = randomSelector(weaponsArray);
+
+  var mistery = [randomCharacter, randomWeapon, randomRoom];
```

Comment:
Let's not make habit of skipping the step where we push objects into arrays. You can do your way since you have 3 objects, but what if you would have 30 or 300 objects.
```
var mistery = [];
mistery.push(randomCharacter, randomWeapon, randomRoom);
```